### ISSUE TITLE
Completely hallucinated web search result

### ISSUE THREAD

**Issue by claell** [huggingface/chat-ui#365]
![grafik](https://github.com/huggingface/chat-ui/assets/6171135/323d28be-cdb2-4229-9a59-32a2e2adceb6)

Chat can be shared upon request.

**Comment by nsarrazin** [comment]
Seems like the summarization model we use is really not that good. We've flagged this before so we'll have to remove it I guess.

**Comment by claell** [comment]
Mh, I hope this can just be improved instead of removed? Maybe there is also some bug happening. This was in a long conversation where the model itself started hallucinating a lot.

**Comment by claell** [comment]
That might be unrelated, though. In general, I just hope, this can be improved instead of removed.

**Comment by nsarrazin** [comment]
Yes of course, I meant that we have to find a better model for summarization and remove this one :stuck_out_tongue:

**Comment by claell** [comment]
Ah, perfect haha

**Comment by claell** [comment]
I don't know how the internal processes work, but why do we need a separate model for summary at all? Can't we just feed the result(s) to the chat model and let it summarize them by itself, then possibly use that as input for the answer to the original question?

**Comment by julien-c** [comment]
(that's what we did initially)
